mysql/src/main/scala/zio/sql/mysql/MysqlSqlModule.scala

```diff
@@ -28,14 +28,14 @@ trait MysqlSqlModule extends Sql { self =>
 
   object MysqlFunctionDef {
     val BitLength   = FunctionDef[String, Int](FunctionName("bit_length"))
+    val Crc32     = FunctionDef[String, Long](FunctionName("crc32"))
     val CurrentDate = Expr.ParenlessFunctionCall0[LocalDate](FunctionName("current_date"))
-    val Crc32       = FunctionDef[String, Long](FunctionName("crc32"))
-    val Degrees     = FunctionDef[Double, Double](FunctionName("degrees"))
-    val Log2        = FunctionDef[Double, Double](FunctionName("log2"))
-    val Log10       = FunctionDef[Double, Double](FunctionName("log10"))
+    val Degrees   = FunctionDef[Double, Double](FunctionName("degrees"))
+    val Log2      = FunctionDef[Double, Double](FunctionName("log2"))
+    val Log10     = FunctionDef[Double, Double](FunctionName("log10"))
     val MakeTime    = FunctionDef[(Int, Int, Double), LocalTime](FunctionName("maketime"))
-    val Now         = FunctionDef[Any, ZonedDateTime](FunctionName("now"))
```

Comment:
did you remove `now` ?

Reply:
good catch a merge mistake. it is fixed now